Templates/ManualTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "defaultFunctionName": "ManualTriggerCSharp",
+    "description": "A C# function that is triggered manually via the portal \"Run\" button",
+    "name": "ManualTrigger - C#",
+    "trigger": "MaunualTrigger",
```

Comment:
Don't know where it got misspelled, I tested manually before sending the PR. There is no automation setup for manual triggers at the moment, also the tests run after the PR has merged in.